mm/ktsan/stack.c

```diff
@@ -14,7 +14,6 @@ void kt_stack_print(kt_stack_t *stack, uptr_t top_pc)
 		pc = kt_decompress(stack->pc[i]);
 		pr_err(" [<%p>] %pS\n", (void *)pc, (void *)pc);
 	}
-	pr_err("\n");
```

Comment:
> All other kt_stack_print are followed by pr_err("\n") in report.c

We need to remove them.

> And that trailing newline before ====== doesn't look good, imho.

It does not look good, but it is better than missed newlines and double newlines if manually sprinkle pr_err("\n") throughout the code.
We can add a newline before ThreadSanitizer: data-race for consistency.